WordPress/src/main/java/org/wordpress/android/ui/avatars/AvatarViewHolder.kt

```diff
@@ -1,7 +1,7 @@
 package org.wordpress.android.ui.avatars
 
 import android.view.ViewGroup
-import org.wordpress.android.databinding.AvatarItemBinding
+import org.wordpress.android.databinding.MySiteBloggingPromptCardAvatarItemBinding
```

Comment:
⚠️ The `AvatarViewHolder` is used in the `TrainOfAvatarsAdapter` which is used to display post likes in the Reader detail screen. I think if we are going to have a specific `ViewHolder` for the avatar shown in the `MySiteBloggingPromptCard` then it should be a whole different `ViewHolder` that we can set the Adapter to use in that specific scenario. (see the other comment showing the changes in reader)